### ISSUE TITLE
Choose genomes for workshop

### ISSUE THREAD

**Issue by widdowquinn** [sipbs-compbiol/BM327-microbiology#2]
We want to choose, e.g. pathogen vs non-pathogen genomes.

- how many?
- which organisms?
- are we really discriminting pathogens or fakey "pathogens" on the basis of MLST?

**Comment by mafeeney** [comment]
Some thoughts on organism choice (based on course alignment with BM329 material) - they cover: _S. aureus, Mycobacterium, Streptococcus, C. diff, Salmonella, Shigella,_ uropathogenic _E. _coli_ (also some viruses and fungi, but I assume we are going to focus on bacterial genomes here, maybe? (or not!))

It would be more work for us, but I wonder if it would be possible to do it as a discovery-based approach (sort of like a CURE), where the students are addressing a real research question (not just rote repetition of a bioinformatic task that feels kind of stale and boring) - maybe by picking a different set of organisms every year....?

Also more work, but what about an idea of letting the students vote for which organism they'd like to choose? (We could make it more focussed by giving a multiple-choice poll, rather than an open-ended question?) 

Would we want to make it a collaborative project in a way (e.g., pick 100 genomes and each student analyses 5 - then compare/compile results?)

Are we really talking about genomes, or are we talking about chromosomes? (Think it might matter a lot for e.g., Bacillus pathogens - probably many other examples too....)

**Comment by widdowquinn** [comment]
> I assume we are going to focus on bacterial genomes here, maybe?

Seems the easiest route, to me. Fungi are complicated, and there are fewer genomes to pick from, in general.

> I wonder if it would be possible to do it as a discovery-based approach (sort of like a CURE)

If we/they chose a set of organisms that did not have a useful marker-based diagnostic, and it turned out to be useful, then yes. In plant pathology, there are a whole bunch of such likely candidates *but* doing this in a way that has impact int he outside world has additional steps: use of all available gneome information; _in vitro_ validation against panels from culture collections; and ring tests to establish consistency.

Where this might come in, is in generating initial candidates to take forward with that more robust screening. If we could partner up with, say, the clinical microbiology lab somewhere, or SASA (in Scotland) or another national body who might be interested in the diagnostic, it's possibly got some mileage.

> what about an idea of letting the students vote for which organism they'd like to choose?

I think multiple-choice here, rather than free choice. I did something similar for last year's UG projects.

> Are we really talking about genomes, or are we talking about chromosomes? (Think it might matter a lot for e.g., Bacillus pathogens - probably many other examples too....)

I think genomes. If your diagnostic target is on a chromosome, then fair enough. If it's on an extrachromosomal element, then we need to question why this target was shared by all of our (pathogenic?) ingroup, but not the closely-related outgroup. If we're diagnosing pathogenicity, rather than chromosomal lineage, then we're probably doing the right thing.

**Comment by mafeeney** [comment]
Yep, I was just concerned in case there might be some cases where the plasmid sequence(s) were lost/not assembled - hopefully that won't be a problem.

Okay, so - multiple choice - I think it would be nice to align with the current class format for BM329 but expand the scope beyond the pathogens that they've already seen in class. 

block B - diseases of the circulatory system and skin
block C - respiratory tract
block D - intestinal tract
block E - urinal tract

So, what about:
_Cutibacterium acnes
Pseudomonas aeruginosa
Campylobacter jejuni
Proteus mirabilis_

Or, do you think they'd get more out of it if we stuck with the ones listed above (that they've already seen in lecture material)? 
_

**Comment by widdowquinn** [comment]
> Yep, I was just concerned in case there might be some cases where the plasmid sequence(s) were lost/not assembled - hopefully that won't be a problem.

I think this is undoubtedly the case in some genomes, especially the more fragmented ones. There may be a tentative identification of plasmid vs chromosomal sequence (e.g. by GC% or similar) but my sense is that this is not routinely annotated.

> So, what about:
> _Cutibacterium acnes Pseudomonas aeruginosa Campylobacter jejuni Proteus mirabilis_

I'm familiar with Pseuds and Campy - not at all with Cutibacterium, and only noddingly with Proteus. That shouldn't stop us, though - it just prevents me having an informed opinion ;)

**Comment by mafeeney** [comment]
quite happy to pick any others if you'd like, that was just me spitballing some initial ideas!

**Comment by widdowquinn** [comment]
I'm happy to be boring for UTIs and pick _E.coli_ - if my old friend and colleague Frank from the Hutton was still around I could have asked him for some ideas - he had lots of experience with UTIs and could diagnose by odour, in the end. Aren't there some Pseuds that cause skin infections ("Hot Tub Folliculitis"?) - might be interesting to contrast with P. aeruginosa?

**Comment by widdowquinn** [comment]
But I'm all for learning about new things - your choices are good.

**Comment by mafeeney** [comment]
We did switch our MLVA typing to lab 5 this year (because of boring scheduling reasons) - and looked at doi: [10.1128/AEM.02812-18](https://dx.doi.org/10.1128%2FAEM.02812-18) ... not 100% sure we'll keep that change for next year (we had previously done the MLVA on _S. aureus_) but if so, it would mesh really nicely with choosing UPEC for this exercise!

I think _P. aeruginosa_ does also cause skin infections (not sure about the hot tub part but it seems likely) - idk how much other species might do as well. Might be a bit of a messy choice with lots of sampling bias (then again, that will likely be a problem for most pathogens we might pick).

**Comment by widdowquinn** [comment]
> I think _P. aeruginosa_ does also cause skin infections (not sure about the hot tub part but it seems likely) - idk how much other species might do as well. Might be a bit of a messy choice

Aye - _Pseud_s have some taxonomy issues, too - depending on which group you're looking at. A nice control might be a plant pathogen (e.g. _P. savastanoi_?). Is _Cutibacterium_ skin-specific? If so, it's probably the better choice.

> looked at doi: [10.1128/AEM.02812-18](https://dx.doi.org/10.1128%2FAEM.02812-18)

Interesting… Maybe a stretch goal of "predict the MLST/MLVA/VNTR profiles for your isolates and see how well they match your diagnostic primer separation" might be an interesting thing to include?

**Comment by mafeeney** [comment]
sounds good! Okay, options are 
_Cutibacterium acnes
Pseudomonas aeruginosa
Campylobacter jejuni_
 & UPEC

I'll start having a look for some genomes - you reckon maybe ~20 each? (20 path + 20 non-path)?

**Comment by mafeeney** [comment]
Right, run into the (entirely predictable) problem of genomes not having enough associated metadata for me to be able to tell if they're from a path/non-path strain (what does that even mean.) Zoom to discuss at some point, maybe?
